pkg/schemaconv/testdata/preserve-unknown.yaml

```diff
@@ -2,31 +2,15 @@ types:
 - name: io.k8s.testcase.Empty
   map:
     elementType:
-      scalar: untyped
-      list:
-        elementType:
-          namedType: __untyped_atomic_
-        elementRelationship: atomic
-      map:
-        elementType:
-          namedType: __untyped_deduced_
-        elementRelationship: separable
+      namedType: __untyped_deduced_
```

Comment:
wasn't sure if it would break other tests.

it did, but I added a leading commit which regenerated expected result `new_schema.yaml` which resolved it. swagger_1.26 is now removed